adapters/huaweiads/huaweiads.go

```diff
@@ -771,7 +771,7 @@ func convertCountryCode(country string) (out string) {
 		return mappedCountry
 	}
 
-	if len(country) >= 3 {
+	if len(country) >= 2 {
```

Comment:
could you please add the test case for this?

Reply:
@clho40  could you address this comment to move forward with this PR

Reply:
Hello,

Just did. Thanks!